cypress/cypress/integration/header.spec.ts

```diff
@@ -182,6 +182,44 @@ describe("Header", () => {
   });
 
   it("shows full branding on desktop if config.styleHeaderColor, config.styleHeaderTextColor, and config.styleHeaderLogo are set", () => {
+    mockDefaultSetup(cy, {
+      config: {
+        cmi5Enabled: false,
+        mentorsDefault: ["clint", "carlos"],
+        styleHeaderColor: "#990000",
+        styleHeaderTextColor: "#FFFFFF",
+        styleHeaderLogo:
+          "http://scribe.usc.edu/wp-content/uploads/2021/02/PrimShield_Word_SmUse_Gold-Wh_RGB-1.png",
```

Comment:
so you're doing this right in the sense that you have the url mocked w intercept below, but it's confusing, because a reader is going to wonder whether this is mocked.

I would clarify by having a const at the top of this spec and then using the const, eg.

```typescript
const FAKE_STYLE_HEADER_LOGO = "http://fake.com/styleheaderlogo.png"

//...
mockDefaultSetup(cy, {
      config: {
        styleHeaderLogo: FAKE_STYLE_HEADER_LOGO
         //...
      }
})

```

Reply:
OK, I copied these tests for `header.spec.ts`. Do you want me to also fix the ones on `header.spec.ts`, too?

Reply:
yes, and generally don't reply to comments in github, that you did a thing until you have pushed the commit

Reply:
https://github.com/mentorpal/mentor-client/pull/84/commits/477457ddff3b924389d75f8c33b1cf9db2871651